app/models/page_redirect.rb

```diff
@@ -0,0 +1,25 @@
+class PageRedirect < ApplicationRecord
+  SOURCES = %w[admin service].freeze
+
+  validates :old_path, presence: true, uniqueness: true
```

Comment:
the uniqueness index here doesn't correspond to the uniqueness index built in the migration (well, depends on which of the unique indexes you decide to keep)

Reply:
I believe this corresponds to https://github.com/thepracticaldev/dev.to/pull/8161/files#diff-1acd2e7e27a227829d5d14a91c863bb6R784 which I plan to keep.

Reply:
Sure, but the migration file has 2 unique indexes, I think the one that's scoped to two columns is redundant